test/integration/app.js

```diff
@@ -72,7 +72,7 @@ describe('app', () => {
           expect(contactDetailsText).to.include('Reception: 01323 502200');
           expect(contactDetailsText).to.include('Fax: 01323 500527');
           expect($('.gp-email').text()).to.include('admin.parkpractice@nhs.net');
-          expect($('.gp-website').text()).to.include('www.parkpractice.co.uk');
```

Comment:
It would be good to add a test for the href to check the link is 'proper'. e.g.
`expect($('.gp-website').prop('href')).to.equal('http://www.parkpractice.co.uk');`

Also, can this test be changed to check the text is `equal` rather than `include` please? And the same for the email address? And the contact details? Ta.